docs/development/testguide.rst

```diff
@@ -143,63 +161,69 @@ Astropy Test Function
 
 .. autofunction:: astropy.test
 
-pytest
-======
+tox
+===
 
-The test suite can be run directly from the native ``pytest`` command. In this
-case, it is important for developers to be aware that they must manually
-rebuild any extensions by running ``setup.py build_ext`` before testing.
+Finally, the most robust way to run the tests (which can also be the slowest) is
```

Comment:
Again, the preferred way should be mentioned first, or at least, link to this section from the `pytest` section above saying something like "but go to this section for the recommended way to run tests" or something like that.

Reply:
I moved tox to be first and edited the text accordingly